src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected static int numberOfTasks = 0;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        numberOfTasks++;
+    }
+
+    public String getDescription(){
+        return this.description;
+    }
+
+    public void setDone(boolean isDone){
+        this.isDone = isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public int getNumberOfTasks(){
+        return numberOfTasks;
+    }
+
+    public String toString(){
```

Comment:
Since all classes in Java inherit from the Object class (directly/indirectly) it would be a better practice to include @\Override annotation when overriding the toString() method because the default toString method of the Object class prints [ObjectName] + Hex Code.

Reply:
will update, thanks